rush-plugins/rush-init-project-plugin/src/logic/template.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Template config File",
+  "description": "The main configuration file Template management.",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "templates": {
+      "description": "A list of templates managed by this tool.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "displayName": {
+            "description": "The display name of the template.",
+            "type": "string"
+          },
+          "folderName": {
+            "description": "The path to the template folder relative to the template config file.",
+            "type": "string"
+          },
+          "description": {
```

Comment:
What's the usage of `description`, `tags`, `author` ?

Reply:
`description`, `tags`, `author` might be used in the future, I will remove it now.